design_handoff_kashakeibo/_ds/organic-dc1d89a9-d628-41b7-996e-f57cf805588b/styles.css

```
@@ -0,0 +1,257 @@
+/* Organic — design-system tokens and component classes. This file is the source of truth for the system's look; retune it here and see readme.md. */
+@import url('https://fonts.googleapis.com/css2?family=Caprasimo:wght@400&family=Figtree:wght@400;600;700&display=swap');
```

Stylelint (17.14.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Caprasimo:wght@400&family=Figtree:wght@400;600;700&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Caprasimo:wght@400&family=Figtree:wght@400;600;700&display=swap'" (import-notation)

(import-notation)